### DISCUSSION TITLE
Soft Assertions

### DISCUSSION THREAD

**Discussion by letaoana** [leancodepl/patrol#1461] [Q&A]
Does the framework support soft assertions?

E.g. https://docs.nunit.org/articles/nunit/writing-tests/assertions/multiple-asserts.html

**Comment by bartekpacia** [comment]
Hi @letaoana 👋🏻 

No, we don't have this feature right now, but we're tracking it in #201. Feel free to comment on your use case there so we can better understand it when we get to implement it someday :)
